packages/base/src/asset-registries/Illustrations.js

```diff
@@ -13,18 +14,58 @@ const registerIllustration = (name, { dialogSvg, sceneSvg, spotSvg, set, title,
 	});
 };
 
+const registerIllustrationLoader = async (illustrationName, loader) => {
+	loaders.set(illustrationName, loader);
```

Comment:
This `registerIllustrationLoader` is called from the generated code and one thing that I think will make code easier to read would be to register the loader to the same key as the key you later try to get in `_loadIllustrationCollectionOnce` within these lines:

```js
const loadIllustrations = loaders.get(illustrationName);
return loadIllustrations(temp);
```

Then, you won't need to get the loader by one key and call `loadIllustrations` with another (with and without "Tnt" I guess) and it will become straightforward and less room for error - using the same key everywhere.

`loaders.set(illustrationName, loader);`

and

`const loadIllustrations = loaders.get(illustrationName);`
`return loadIllustrations(illustrationName);`

Reply:
I made them with separted keys, because when an illustration from the Tnt collection is demanded, the user demand it with Tnt prefix, but the illustraiton itself, doesnt have Tnt prefix in it, for example: 
- User creates an illustrated message with attribute name: TntChartArea, but the illustration name is only ChartArea. 

This is the workaround I came up with, to implement this part of the change.

Reply:
What I meant was if you could try changing the loader registration. In the generated code (dist/generated/js-imports/Illlustration.js), where you register loaders, you can append "Tnt" to those tnt names below, because you know there are in the "tnt" folder:

```js
["ChartArea", "ChartArea2", "ChartBPMNFlow", "ChartBar", "ChartBullet", "ChartDoughnut", "ChartFlow", "ChartGantt", "ChartOrg", "ChartPie", "CodePlaceholder", "Company", "Components", "ExternalLink", "FaceID", "Fingerprint", "Lock", "Mission", "NoApplications", "NoFlows", "NoUsers", "Radar", "Secrets", "Services", "SessionExpired", "SessionExpiring", "Success", "SuccessfulAuth", "Systems", "Teams", "Tools", "UnableToLoad", "Unlock", "UnsuccessfulAuth", "User2"].forEach(illustrationName => registerIllustrationLoader(illustrationName, loadAndCheck));
```

Then, without any temp variables, without any checks, you will simply can:

```js
const loadIllustrations = loaders.get(illustrationName);
return loadIllustrations(illustrationName);
```